railties/lib/rails/generators/rails/app/templates/config/databases/ibm_db.yml

```diff
@@ -11,14 +11,14 @@
 #       export IBM_DB_LIB=/opt/ibm/db2/V9.7/lib32
 #       gem install ibm_db
 #
-#   On Mac OS X 10.5:
+#   On macOS 10.5:
```

Comment:
Yeah, good point. Same as above, I feel like we should try and have one way to refer to the OS though, to avoid confusion